### ISSUE TITLE
Error: Reveal.availableFragments is not a function

### ISSUE THREAD

**Issue by Zhen-hao** [astefanutti/decktape#185]
Reveal JS plugin activated
Printing slide #/slide-9 ( 12/ ?) ...
Error: Evaluation failed: TypeError: Reveal.availableFragments is not a function
    at __puppeteer_evaluation_script__:1:39

I am on macOS 10.14.5 and chrome version 74.0.3729.169.
To reproduce, clone [this repo](https://github.com/Zhen-hao/fp-ams-meetup-march-2019) and run `decktape automatic index.html  prentation.pdf` in it

**Comment by astefanutti** [comment]
The Reveal plugin is compatible with Reveal version 2.4+. It seems you're using version 2.3.0, which is quite old. Could you try upgrading your Reveal lib? On my side, I'll update the existing error message, which is displayed for version < 2.3.0.

**Comment by astefanutti** [comment]
I've just updated the compatibility check in d16dbbe7254378f4e076248e63958d7601524de6.

**Comment by Zhen-hao** [comment]
Hi Antonin
Thanks for the quick response. 
I updated the reveal.js version to 3.8. Now I still have the error, among other ones.

Loading page file:///Volumes/MacP2/workspace/presentations/fp-ams-meetup-march-2019/index.html ...
WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips.

Page error: TypeError: Reveal.registerPlugin is not a function
    at file:///Volumes/MacP2/workspace/presentations/fp-ams-meetup-march-2019/frameworks/revealjs/plugin/zoom-js/zoom.js:30:8

Page error: TypeError: Reveal.registerPlugin is not a function
    at file:///Volumes/MacP2/workspace/presentations/fp-ams-meetup-march-2019/frameworks/revealjs/plugin/notes/notes.js:178:8
Loading page finished with status: 0
Reveal JS plugin activated
Printing slide #/slide-9 ( 12/ ?) ...
Error: Evaluation failed: TypeError: Reveal.availableFragments is not a function
    at __puppeteer_evaluation_script__:1:39

I updated the presentation repo. can you try to reproduce it? 

Many thanks,
Zhen

**Comment by astefanutti** [comment]
I've just tried loading your presentation in a Web browser and the errors occur as well, which indicate its not related to Decktape:

```
MathJax.js?config=TeX-AMS-MML_HTMLorMML:32 WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips.
replaceScript @ MathJax.js?config=TeX-AMS-MML_HTMLorMML:32
zoom.js:30 Uncaught TypeError: Reveal.registerPlugin is not a function
    at zoom.js:30
notes.js:178 Uncaught TypeError: Reveal.registerPlugin is not a function
    at notes.js:178
```

It may be the upgrade from 2.3.0 to 3.8.0 requires some adjustments in the way you bootstrap your presentation.

**Comment by Zhen-hao** [comment]
It's hard for me to tell what's missing because I can open the index.html with any browser and the slides work fine. Any tips?

**Comment by astefanutti** [comment]
> It's hard for me to tell what's missing because I can open the index.html with any browser and the slides work fine. Any tips?

I think you've forgotten to upgrade the `framworks/revealjs/js/reveal.min.js` file that is actually loaded in your presentation.

**Comment by Zhen-hao** [comment]
Thank you very much! 
Turned out I need to update some files in the partials folder and layout folder. I updated the repo with the fix. 
Now I can run the command. 
But the only issue now is the alignment is not right. See the screenshot.

<img width="1072" alt="Screenshot 2019-06-07 at 23 58 58" src="https://user-images.githubusercontent.com/10957195/59135727-5c5e1680-8980-11e9-8464-953770951adc.png">
